doc/style-guide.md

```diff
@@ -44,3 +45,52 @@ quaility.
     character limit.
 
 **[top](#table-of-contents)**
+
+## Expressions
+
+  - Expressions in return statements should be surrounded by parenthesis for clarity.
+    - If the expression contains one operand, parenthesis are not necessary.
+    - Use surrounding parenthesis around the conditional expression of a *ternary
+      operator*; not for the entire ternary expression.
+  ```typescript
+  // bad
+  function foo(a: number) {
+      return a > 1;
+  }
+
+  // bad
+  function foo(a: number) {
+      return a > 1 ? true : false;
+  }
+
+  // OK
+  function foo(a: number) {
+      return ((a > 1) ? true : false);
+  }
+
+  // OK
+  function isZero(a: number) {
+      return (!a);
+  }
+
+  // good
+  function foo(a: number) {
+      return (a > 1);
+  }
+
+  // good
+  function foo(a: number) {
+      return (a > 1) ? true : false;
```

Comment:
Maybe change the return values to non-boolean. In this example, I agree with the stylistic issue here of where the parens go, but I'd complain that you shouldn't use a ternary operator because you should just be returning (a > 1). :)